ui/src/views/DnsBlacklist.vue

```diff
@@ -294,7 +297,7 @@
                 >
                   <span
                     :class="['confidence', {'green': (props.row.enabled && config.status && props.row.confidence > 6)},
-                    {'orange': (props.row.enabled && config.status && props.row.confidence <= 6)}]"
+                    {'orange': (props.row.enabled && config.status && props.row.confidence <= 6)},{'orange': (props.row.missing && config.status)}]"
```

Comment:
Maybe this can be rewritten more concisely:
```
{'orange': config.status && (props.row.enabled && props.row.confidence <= 6 || props.row.missing)}
```

Reply:
agreed much better